rust/cache/src/foyer.rs

```diff
@@ -316,21 +350,25 @@ where
 {
     #[tracing::instrument(skip(self, key))]
     async fn get(&self, key: &K) -> Result<Option<V>, CacheError> {
+        let _stopwatch = Stopwatch::new(&self.get_latency);
         Ok(self.cache.get(key).await?.map(|v| v.value().clone()))
     }
 
     #[tracing::instrument(skip(self, key, value))]
     async fn insert(&self, key: K, value: V) {
+        let _stopwatch = Stopwatch::new(&self.insert_latency);
```

Comment:
curious why this is measured separately instead of looking at the span duration?